mlrun/frameworks/sklearn/__init__.py

```diff
@@ -180,3 +181,15 @@ def apply_mlrun(
     )
 
     return handler
+
+
+# TODO: Remove in MLRun version 1.2.0.
```

Comment:
In order to be backwards compatible we need to be backwards compatible 6 months back.
```suggestion
# TODO: Remove once 1.0.0 is no longer supported
```

Reply:
changed.